packages/next/README.md

```diff
@@ -2627,9 +2627,8 @@ You can use [now dev](https://zeit.co/docs/v2/development/basics) as your local
     { "src": "home/next.config.js", "use": "@now/next" }
   ],
   "routes": [
-    { "src": "/docs/_next(.*)", "dest": "docs/_next$1" },
-    { "src": "/docs(.*)", "dest": "docs/docs$1" },
-    { "src": "(.*)", "dest": "home$1" }
+    { "src": "/docs(.*)", "dest": "docs$1", "continue": true },
+    { "src": "(.*)", "dest": "home$1", "continue": true }
```

Comment:
```suggestion
    { "src": "(?!/?docs)(.*)", "dest": "home$1", "continue": true }
```

This needs to be a negative lookahead because of the `continue` AFAIK, can you test @paulogdm?

Reply:
Cc @dav-is

Reply:
I think this is a check you shouldn't have to do given the previous route.

Reply:
@timer is right unless you structured everything like this:
```
src/
  docs/
  ...files
```